src/components/Filter/types/index.ts

```diff
@@ -73,9 +73,12 @@ export interface FilterModule<T> {
      */
     getFilterBarLabel(value: T): string;
     /**
-     * Generates the array of values to be displayed on this filter's section in the filter pane.
+     * Generates the array of labels to be displayed on this filter's section in the filter pane.
      */
     getFilterSectionLabel(value: T): string | string[];
+    /**
```

Comment:
I believe these comments are no longer needed (and/or aren't being used as originally expected). Please remove.